demo/ServiceHook.fs

```diff
@@ -0,0 +1,62 @@
+module Demo.ServiceHook
+
+open Feliz
+open Browser
+open Fable.Core.JsInterop
+open Glutinum.Feliz.ReactGoogleAutocomplete
+open type Glutinum.Feliz.ReactGoogleAutocomplete.Exports
+open Fable.Core.JS
+
+[<ReactComponent>]
+let private Component () =
+    let response =
+        usePlacesAutocompleteService (
+            UsePlacesAutocompleteServiceConfig(
+                apiKey = VITE_GOOGLE_MAP_API_KEY,
+                debounce = 300
+            )
+        )
+
```

Comment:
The original exemple from https://github.com/ErrorPro/react-google-autocomplete#useplacesautocompleteservice is using a `useEffect`, if needed we can also add it.